core-profile-tck/tck-dist/README.adoc

```diff
@@ -0,0 +1,44 @@
+= Jakarta Core Profile TCK
+
+This is the TCK distribution for the Jakarta Core Profile.
+
+This distribution, as a whole, is licensed under the terms of the Apache Public License, while the final distribution used for compatibility testing is under the EFTL. The bundled LICENSE file will dictate which license applies to the TCK distribution.
+
+This distribution consists of:
+
+artifacts/
+
+ – TCK binaries and sources, packaged as jars
+ – TestNG suite.xml file for running the CDI Lite TCK
+ - maven pom.xml to install artifacts into a local repo
+
+doc/
+
+ – A detailed reference guide for the current version of TCK in both pdf and html formats
+
+
+examples/
+
+- example maven based TCK runner projects for compatible implementations
+
+== Software Requirements
+
+* A Java SE 11 runtime
```

Comment:
I suggest not to limit to Java SE 11. We should say Java SE 11 - 17 runtime. Not sure about Java 18 as yet due to SecurityManager removal.

Reply:
Updated to say 11+